ci/setup_env.sh

```diff
@@ -36,9 +43,24 @@ else
   exit 1
 fi
 
-wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh
-chmod +x miniconda.sh
-./miniconda.sh -b
+if [ `uname -m` = 'aarch64' ]; then
+   wget -q "https://github.com/Archiconda/build-tools/releases/download/0.2.3/Archiconda3-0.2.3-Linux-aarch64.sh" -O archiconda.sh
+   chmod +x archiconda.sh
+   $IS_SUDO apt-get install python-dev
+   $IS_SUDO apt-get install python3-pip
+   $IS_SUDO apt-get install lib$ARCHICONDA_PYTHON-dev
+   $IS_SUDO apt-get install xvfb
+   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib:/usr/local/lib:/usr/local/bin/python
+   ./archiconda.sh -b
+   echo "chmod MINICONDA_DIR"
+   $IS_SUDO chmod -R 777 $MINICONDA_DIR
+   $IS_SUDO cp $MINICONDA_DIR/bin/* /usr/bin/
+   $IS_SUDO rm /usr/bin/lsb_release
```

Comment:
Just leave the wget here, and save the file as `miniconda.sh`, so you can move the chmod and the call to it outside of the if. W edon't need the installs or anything else